app/assets/stylesheets/publisher.scss

```diff
@@ -243,14 +237,6 @@
     .hide-location { display: inline-block; }
     .locator { display: none; }
   }
-
-  .twitter-typeahead {
-    left: -1px;
-    // Override inline rule of Typeahead
-    // scss-lint:disable ImportantRule
-    position: absolute !important;
```

Comment:
Oh, width suffers from exactly the same problem: if the box is absolutely positioned, then it's width is defined by the closest relatively positioned parent. If this parent is outside the comment input box, the we have wrong width. If it is inside the comment input box, then suggestions box get cut because of the `overflow: hidden` setting of the comment widget. Maybe it can be solved by removing `overflow: hidden` (there is no `overflow: hidden` property used in the post publisher), but it breaks the layout this way, so this must be solved also.